### PR TITLE
adds project to provision custom nodes

### PR DESCRIPTION
**Pull request by ankul-shippable**
https://github.com/Shippable/pm/issues/8260

```
ankul@ankul:~/Desktop/shippable/provision-custom-nodes(master)$ node createCustomNode.js 
Reading config.json file
creating clusterNode
Initializing ShippableAdapter with token: 123547ab-4828-49fa-a4d7-7f0071135abc
getting node initialization script
saving initScript in initScript.sh
successfully created clusterNode

```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "provision-custom-nodes",
+  "license": "UNLICENSED",
+  "dependencies": {
+    "async": "^2.2.0",
```
Comment (himanshu0503) [inline package.json]: Let's hardcode the dependencies.

File: createCustomNode.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    function (err) {
+      if (err)
+        return next(err);
+
+      return next();
+    }
+  );
+}
+
+// call createClusterNode
```
Comment (himanshu0503) [inline createCustomNode.js]: This can be removed.

File: README.md [hunk 3]
```diff
@@ -1 +1,20 @@
...
+Please follow these steps to provision a custom node-
+
+1. clone this project
+2. update `config.json` to have correct values.
+
+    a. `SHIPPABLE_API_TOKEN` is your apiToken.
+
+    b. `FRIENDLY_NAME` is the name of your clusterNode.
+
+    c. `IS_SHIPPABLE_INITIALIZED` should be set to `false`, if you want to run the node initialization script yourself. If it is set to `true` shippable will run the script for you, but before provisioning the node you have to run the command on your node to create a shippable user and ssh authorization(This command is available in your subscriptions/settings/nodes, click on `add node` and copy the command).
```
Comment (himanshu0503) [inline README.md:12]: We should add a screenshot of the add key.

Comment (ankul-shippable) [reply README.md:12]: okay, updating

**Pull request merged**